src/guides/resolve-zil-domain.md

```diff
@@ -0,0 +1,244 @@
+---
+description: Simplest step-by-step guide on how to resolve .zil domain
+---
+
+# .zil Integration
+
+In order to resolve a .zil domain, we will need to satisfy three steps below
+
+* Get a namehash of a domain
+* Get resolver contract address from a domain
+* Query resolver contract to fetch the records
+
+Let's visualize the resolution process using some of the simplest tools web developer has: knowledge of HTML and js.
+
+### Initialize the project folder
+
+As has been said above all we need is to create a folder and two files index.html and index.js respectively
+
+```text
+mkdir unstoppable-zil-resolution
+cd unstoppable-zil-resolution
+touch index.html index.js
+```
+
+Let's use some blank HTML page. We have connected our empty index.js as well as a CDN library of [**js-sha256**](https://www.npmjs.com/package/js-sha256?utm_source=cdnjs&utm_medium=cdnjs_link&utm_campaign=cdnjs_library) for future namehashing step.
+
+```markup
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Basic .zil integration</title>
+    </head>
+    <body>
+
+      <script 
+        src="https://cdnjs.cloudflare.com/ajax/libs/js-sha256/0.9.0/sha256.min.js" 
+        integrity="sha512-szJ5FSo9hEmXXe7b5AUVtn/WnL8a5VofnFeYC2i2z03uS2LhAch7ewNLbl5flsEmTTimMN0enBZg/3sQ+YOSzQ==" 
+        crossorigin="anonymous"></script>
```

Comment:
We should either use 4 spaces here or 2 spaces indentation in other parts to be consistent.